### DISCUSSION TITLE
Add product preview for disabled sites

### DISCUSSION THREAD

**Discussion by qbasic16** [craftcms/commerce#2463] [Ideas]
**What are you trying to do?**
Howdy,
It would be a great enhancement, if you could make the preview work in multi-site mode with sites, that are (for the moment) disabled for frontend.

Often times we implement online shops that start with 1 language and have 1 or 2 additional sites of other languages disabled, until all content is translated and ready for visitors. Editors struggle in the back-end to update translations without a preview.

**What's your proposed solution?**
Make the product preview work as it behaves with Entry previews: They also show a preview for disables sites ("enabled for front-end" turned off).

**Additional context**
For now, we just enable the sites but have to exclude them from sitemap generation (SEOmatic) and in the robots.txt.

May you be praised with the best beers, when this is implemented :)

**Comment by brandonkelly** [comment]
Hey @qbasic16, can you please tell me what your sites’ base URLs are, and which ones aren’t working with Live Preview?

**Comment by lukeholder** [comment]
@qbasic16 any info Brandon as requested?
